### ISSUE TITLE
iOS error: undefined symbols for architecture arm64

### ISSUE THREAD

**Issue by ronherrema** [enzienaudio/heavy#117]
I've been trying to set up the C code from one of my patches in Xcode 7 and run it on my 64 bit Mini iPad. I get the following errors:

![screen shot 2016-08-06 at 19 10 49](https://cloud.githubusercontent.com/assets/3489924/17458505/60ee6cb4-5c0a-11e6-9dd1-256866253903.png)

Can you tell me what might be going on? 

A related request: It would be great if there was an iOS example project we could download.

Thanks

**Comment by mhroth** [comment]
@ronherrema Are you sure that you're including all files in the compile process? Seems like potentially `HvContext_drones_enzien.c` is not included?

About the Xcode example project. That sounds reasonable. What would you want it to do?

**Comment by ronherrema** [comment]
You're right - I had not copied all the files over correctly. It now compiles, but no sound :-/ (I mentioned to Joe in an email that I had tried the javascript version - which runs fine in a browser - within an iOS WebView and that I also had no sound in that case.)

The example project could be a swiss army knife version, including file playback, mic input, and real-time synthesis with interactive controls. But for my purposes at the moment I'm only interested in the last of these, so a simple FM patch, for example, with some touch inputs (could be standard iOS sliders) to control the synthesis parameters.

Thanks

**Comment by mhroth** [comment]
So no sound at all on iOS? Presumably Core Audio is correctly configured? What's the enzien link for the patch? I can take a look.

**Comment by ronherrema** [comment]
No, no sound. What would 'correctly configured' include? I've just tried adding the CoreAudio and CoreAudioKit frameworks, but that makes no difference.

**Comment by diplojocus** [comment]
@ronherrema can you paste your code in a gist or try uploading the project?

Generally with CoreAudio there's a bit of boilerplate to set up in order to interface with the audio drivers.

**Comment by mhroth** [comment]
Ahh! Heavy only implements the signal processing, it doesn't manage the audio loop or anything like that. You'd have to set that up yourself using any number of frameworks like [Superpowered](http://superpowered.com/) or [JUCE](https://www.juce.com/), etc.

Either way, I'll look into getting an iOS demo app to you.

**Comment by ronherrema** [comment]
OK - great. The link, by the way, is this:

https://enzienaudio.com/h/ronherrema/drones_enzien/

Joe - I've sent you the javascript project in an email, and I'll send this one over as well.

Thanks
